front/app/modules/commercial/custom_maps/admin/containers/ProjectCustomMapConfigPage/addOrderingToLayers.ts

```diff
@@ -0,0 +1,36 @@
+import { IMapLayerAttributes } from '../../../services/mapLayers';
```

Comment:
Isn't this something the back-end should handle?

Reply:
Yes, this is a bug. I will log to add this to the backend